templates/results-row.php

```diff
@@ -14,5 +14,13 @@
 	<td>
 		{{data.message}}
 	</td>
+	<# if ( data.hasLinks ) { #>
+		<td>
+			<a href="{{data.link}}"<?php echo ( ! has_filter( 'wp_plugin_check_validation_error_source_file_editor_url_template' ) ) ? ' target="_blank"' : '' ?>>
+				<?php esc_html_e( 'View in code editor', 'plugin-check' ); ?>
+				<span class="screen-reader-text"><?php esc_html_e( '(opens in a new tab)', 'plugin-check' ); ?></span>
+			</a>
```

Comment:
I think this still needs to be wrapped in a `data.link` if check, to make sure we're not rendering an `<a>` tag that doesn't go anywhere. Only the `<td>` must be rendered regardless of `data.link`.

Reply:
Thanks for the feedback. I have updated in https://github.com/WordPress/plugin-check/pull/298/commits/55d8780ec5afb884020e8f13d22002e50caf4c65